org_status/org_status.py

```diff
@@ -84,6 +84,13 @@ def main():
     for Host, org in generate_fetch_jobs(args.orgs):
         token = None
 
+        try:
+            if not Host.get_status():
+                print(Host.HostName + ' is currently down')
+                continue
+        except NotImplementedError:
+            pass
```

Comment:
Add a similar lambda as that of `styled` for verbose printing i.e something like `verbose: print if args.verbose else lambda *_: None`. And print a message using it here, say `verbose(f'{Host.HostName} does not support checking host status')`.